<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="preButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>190</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mã số người dùng:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giới tính:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dob">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ngày sinh:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editUserButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chỉnh sửa</string>
   </property>
  </widget>
  <widget class="QLabel" name="num">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>201</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="historyDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>541</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>539</width>
      <height>299</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>preButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>image</zorder>
  <zorder>id</zorder>
  <zorder>name</zorder>
  <zorder>sex</zorder>
  <zorder>dob</zorder>
  <zorder>editUserButton</zorder>
  <zorder>num</zorder>
  <zorder>historyDisplay</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
